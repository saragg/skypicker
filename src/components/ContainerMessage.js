import React from 'react';
import './ContainerMessage.css';
import FontIcon from 'material-ui/FontIcon';

const ContainerMessage = (props) => {
  const iconStyle = {
    color: "#999",
    fontSize: "90px"
  }
  return (
    <div className="container-message">
      <FontIcon style={iconStyle} className={props.icon}/>
      <p>{props.message}</p>
    </div>
  )
}

export default ContainerMessage