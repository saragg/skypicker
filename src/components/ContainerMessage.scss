.container-message {
  color: #999;
  font-size: 24px;
  text-align: center;
}