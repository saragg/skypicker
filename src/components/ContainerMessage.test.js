import React from 'react';
import ContainerMessage from './ContainerMessage';
import renderer from 'react-test-renderer';
import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';

test('Renders correctly', () => {

  const tree = renderer.create(
    <MuiThemeProvider>
      <ContainerMessage icon="fa fa-smile-o" message="Hi!" />
    </MuiThemeProvider>
  ).toJSON();

  expect(tree).toMatchSnapshot();
});