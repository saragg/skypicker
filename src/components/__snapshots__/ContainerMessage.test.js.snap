// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
<div
  className="container-message"
>
  <span
    className="fa fa-smile-o"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    style={
      Object {
        "color": "#999",
        "display": "inline-block",
        "fontSize": "90px",
        "muiPrepared": true,
        "position": "relative",
        "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
        "userSelect": "none",
      }
    }
  />
  <p>
    Hi!
  </p>
</div>
`;
